sendmarc.com.protection.json

```diff
@@ -25,14 +25,14 @@
       "spfRules": "%spf_rules%"
     },
     {
-      "groupId": "sts",
+      "groupId": "_mta-sts",
```

Comment:
This change is not it the PR title, so make sure the group ids were not change by accident.